pkgs/development/python-modules/pytorch/default.nix

```diff
@@ -165,7 +192,7 @@ in buildPythonPackage rec {
   #
   # Also of interest: pytorch ignores CXXFLAGS uses CFLAGS for both C and C++:
   # https://github.com/pytorch/pytorch/blob/v1.2.0/setup.py#L17
-  NIX_CFLAGS_COMPILE = lib.optionals (numpy.blas == mkl) [ "-Wno-error=array-bounds" ];
+  NIX_CFLAGS_COMPILE = lib.optionals mklSupport [ "-Wno-error=array-bounds" ];
```

Comment:
As mentioned, drop the `mklSupport` attr from the top-level API, and replace it with a local variable:

```nix
mklSupport = blas.implementation == "mkl";
```